NEWS

```diff
@@ -1,6 +1,8 @@
 Changes from 0.13.0
 
    [ Language Changes ]
+   * Pythonic falsey values (in means of __bool__) can now be spliced
+     by `unquote-splice`
```

Comment:
I can't tell what this means.

A better wording would be "The `unquote-splice` or `~@` form now accepts any false value as empty."